packages/hint-create-element-svg/tests/tests.ts

```diff
@@ -0,0 +1,69 @@
+import generateHTMLPage from 'hint/dist/src/lib/utils/misc/generate-html-page';
+import { getHintPath } from 'hint/dist/src/lib/utils/hint-helpers';
+import readFile from 'hint/dist/src/lib/utils/fs/read-file';
+import { HintTest } from '@hint/utils-tests-helpers/dist/src/hint-test-type';
+import * as hintRunner from '@hint/utils-tests-helpers/dist/src/hint-runner';
+
+const hintPath = getHintPath(__filename);
+
+const generateHTMLPageWithDivTag = (head: string, body: string) => {
+    return generateHTMLPage(head, `<div id="container">${body}</div>`);
+};
+const generateScriptTag = (script: string) => {
+    return `<script type="text/javascript">${script}</script>`;
+};
+const generatePageWithExternalScript = () => {
+    const externalJS = readFile(`${__dirname}/js/testjs.js`);
```

Comment:
Your tests are failing because the test runner is trying to run your external JS file as well. Move this to a `fixtures` folder to ensure it gets ignored.
```suggestion
    const externalJS = readFile(`${__dirname}/fixtures/testjs.js`);
```